src/api.js

```diff
@@ -73,6 +73,12 @@ const getVanAddrs = function(res, args) {
     }
 
     res.setHeader('Content-Type', 'application/json');
+
```

Comment:
Rather than repeating this for each query, it would be better to have a single function to set the appropriate headers.

Reply:
> Rather than repeating this for each query, it would be better to have a single function to set the appropriate headers.

👍🏼